src/main/java/de/hysky/skyblocker/utils/render/SkyblockerRenderLayers.java

```diff
@@ -58,6 +58,9 @@ public class SkyblockerRenderLayers {
 			.texture(new RenderPhase.Texture(texture, TriState.FALSE, false))
 			.layering(RenderPhase.VIEW_OFFSET_Z_LAYERING)
 			.build(false));
+	public static final MultiPhase CYLINDER = RenderLayer.of("cylinder", RenderLayer.DEFAULT_BUFFER_SIZE, false, true, SkyblockerRenderPipelines.CYLINDER, MultiPhaseParameters.builder()
+			.layering(RenderPhase.VIEW_OFFSET_Z_LAYERING)
+			.build(false));
```

Comment:
Do we need a separate render layer and pipeline? Isn't this just triangle strip?